api/core/model_runtime/model_providers/openai_api_compatible/llm/llm.py

```diff
@@ -372,15 +372,9 @@ def create_final_llm_result_chunk(index: int, message: AssistantPromptMessage, f
 
                 if 'delta' in choice:
                     delta = choice['delta']
-                    if delta.get('content') is None or delta.get('content') == '':
-                        if finish_reason is not None:
-                            yield create_final_llm_result_chunk(
-                                index=chunk_index,
-                                message=AssistantPromptMessage(content=choice.get('text', '')),
-                                finish_reason=finish_reason
-                            )
-                        else:
-                            continue
+                    delta_content = delta.get('content', '')
+                    if delta_content == '':
```

Comment:
```suggestion
                    if delta_content is None or  delta_content == '':
```
Add checking None to conditions.
As in https://github.com/langgenius/dify/issues/2267, `delta.get('content', '')` still turns out to be a `None`, which only checks whether `content` key exists in `delta` dict but does not guarantee it's not a None.

Reply:
thanks,done

Reply:
Thanks. LGTM.